### DISCUSSION TITLE
Indirect landuse change emission

### DISCUSSION THREAD

**Discussion by arby1005** [USEPA/supply-chain-factors#31] [Q&A] [answered]
Hi all, 

Thanks for WesIngwersen answering the question that land use change is not included in the EEIO data. I have a follow-up question on this. If there are any indirect land use change emissions, e.g. land use change for soy production for animal feed in the poultry industry, will it be incorporated or not? 

Thank you.

**Comment by WesIngwersen** [comment, marked as answer]
Hi @arby1005.  This is a good question.  Using life cycle assessment terminology, the approach we have used is strictly an _attributional_ approach, which means these types of indirect effects that demand for a product of interest has on other commodities and the related environmental effects this can have are not included.   Trying to account for that indirect land use change would entail a _consequential_ approach.
